docs/src/content/docs/quickstart-guide.mdx

```diff
@@ -1,96 +1,145 @@
+
+   Sign up for a Terrateam account and install the GitHub app in your organization.
+   
+   - $149/mo unlimited with 14-day free trial (no credit card required)
+   - Select the GitHub organizations and repositories containing your Terraform code.
+   
+   <LinkCard
+     title="Sign up for Terrateam"
+     href="https://terrateam.io/signup"
+   />
+   
+   ![GitHub application installation](../../assets/install-terrateam-github.png)
 
-1.  #### New Branch
-
-    Create a new branch against a repository with Terrateam installed.
-
-    ![Terrateam first run branch](../../assets/terrateam-first-run-branch.png)
-
-1.  #### Add Terraform Code
-
-    Create a new file called `main.tf` in any directory in the repository. [Download the code here](https://raw.githubusercontent.com/terrateamio/terrateam-example/main/main.tf).
-
-    ![Terrateam main.tf](../../assets/nibbler-main-tf.png)
-
-1.  #### Pull Request
-
-    Commit and push `main.tf` to your branch and then create a pull request.
-
-    ![Terrateam pull request](../../assets/nibbler-pull-request.png)
-
-1.  #### Terraform Plan
-
-    Terrateam will automatically trigger a `terraform plan`.
-
-    ![Terrateam plan](../../assets/nibbler-terrateam-plan.png)
-
-1.  #### Terraform Apply
+2. **Add the Terrateam GitHub Actions Workflow**
 
-    Comment `terrateam apply` on the pull request. Terrateam will trigger a `terraform apply`.
+   The workflow file enables Terrateam to run Terraform operations within your GitHub Actions environment.
+   
+   - Create a `.github/workflows` directory in your repository if it doesn't exist.
+   - Download and add the [terrateam.yml](https://raw.githubusercontent.com/terrateamio/terrateam-example/main/.github/actions/workflows/default/terrateam.yml) file to this directory.
+   - Commit the file to your default branch (`main` or `master`).
+   
+   ![GitHub Actions Workflow File](../../assets/github-actions-workflow-file-location.png)
+   
+   :::note
+   This workflow file is essential as it allows Terrateam to execute Terraform commands in response to pull request events.
+   :::
 
-    ![Terrateam Apply](../../assets/nibbler-terrateam-apply.png)
+3. **Configure Cloud Provider Authentication (Optional for this guide)**
 
-1.  #### Merge
+   For real infrastructure deployments, you'll need to set up authentication with your cloud provider.
+   
+   <LinkCard
+     title="Cloud Provider Setup"
+     href="/cloud-providers"
+     description="Configure AWS, GCP, Azure or other providers"
+   />
+</Steps>
 
-    Merge the pull request.
+## Your First Terrateam Deployment
 
-    ![Pull Request Merge](../../assets/nibbler-merge.png)
+Let's test Terrateam with a simple example that doesn't require cloud provider credentials.
 
-    :::tip[Apply after merge]
-    [Click here to learn how to apply your changes after merging a pull request.](/advanced-workflows/apply-after-merge)
-    :::
+:::tip[What is a `null_resource`?]
```

Comment:
The null_resource concept may still be unclear for those brand new to Terraform.